### PR TITLE
restore my sanity

### PR DESCRIPTION
**Pull request by xvrl**

### CODE REVIEW

File: build-tools/src/main/resources/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -172,7 +172,7 @@
         </module>
         <module name="AbbreviationAsWordInName">
             <property name="ignoreFinal" value="false"/>
-            <property name="allowedAbbreviationLength" value="1"/>
```
Comment (ewencp) [inline build-tools/src/main/resources/checkstyle/checkstyle.xml]: Do we have longer counter examples than `DB` followed by the capital for the next word? A setting of `2` captures that case.

Comment (xvrl) [inline build-tools/src/main/resources/checkstyle/checkstyle.xml]: done

**Comment by xvrl** [comment]
@ewencp it shouldn't affect any existing APIs, no?

**Comment by xvrl** [comment]
gtm?

**Pull request merged**